drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -101,14 +104,22 @@ def getTimeStamp():
     return ts
 
 # copies log files from a folder that may contain other files to another path
-def copyLogFilesTo(logFiles, filePath, targetFolderPath):
+# log files that are blacklisted are not copied and thus not registered (after metadata extraction)
+def copyLogFilesTo(logFiles, filePath, targetFolderPath, facilityName):
+    # return list of files to remove for this facility or empty list
+    blacklist = blacklistedByFacility.get(facilityName, [])
+    numberOfFiles = len(logFiles)
     for logFile in logFiles:
-        sourcePath = os.path.join(filePath, logFile.getName())
-        shutil.copy2(sourcePath, targetFolderPath)
-        src = os.path.join(filePath, logFile.getName())
-        shutil.copy2(src, targetFolderPath)
+        fileType = logFile.__class__.__name__
+        if fileType in blacklist:
+            numberOfFiles-=1
```

Comment:
```suggestion
            numberOfFiles -= 1
```
Just for readability

Reply:
Maybe it would be more expressive to introduce a new variable above `numIgnoredFiles` and check later `len(copiedContent) != len(logFiles) - numIgnoredFiles`

Reply:
done